ebi_eva_common_pyutils/reference/assembly.py

```diff
@@ -235,13 +235,14 @@ def download_contig_from_ncbi(self, contig_accession, output_file):
         self.info('Downloading ' + contig_accession)
         urllib.request.urlretrieve(url, output_file)
 
-    def download_or_construct(self, genbank_only=False, overwrite=False):
+    def download_or_construct(self, genbank_only=False, overwrite=False, update=False):
         """First download the assembly report and fasta from the FTP, then append any missing contig from
         the assembly report to the assembly fasta."""
-        self.download_assembly_report(overwrite)
-        try:
-            self.download_assembly_fasta(overwrite)
-        except:
-            pass
+        if not update or overwrite:
+            self.download_assembly_report(overwrite)
+            try:
+                self.download_assembly_fasta(overwrite)
+            except:
+                pass
```

Comment:
Maybe I'm just under-caffeinated, but what's the difference in behaviour between `overwrite` and `update`?  If possible, could you add tests that show the difference, and/or a docstring that explains it?